proto/services/tasks_svc/v1/task_template_svc.proto

```diff
@@ -116,43 +87,19 @@ message CreateTaskTemplateSubTaskResponse {
   string id = 1;
 }
 
-///
-/// GetAllTaskTemplatesByCreator
-///
-
-message GetAllTaskTemplatesByCreatorRequest {
-  string created_by = 1; // @gotags: validate:"required"
-  optional bool private_only = 2;
-}
-
-message GetAllTaskTemplatesByCreatorResponse {
-  repeated TaskTemplate templates = 1;
-
-  message TaskTemplate {
-    string id = 1;
-    string name = 2;
-    string description = 3;
-    bool is_public = 4;
-    repeated SubTask subtasks = 7;
-
-    message SubTask {
-      string id = 1;
-      string task_template_id = 2;
-      string name = 3;
-    }
-  }
-}
-
 
 ///
-/// GetAllTaskTemplatesByWard
+/// GetAllTaskTemplatesByWardAndCreatorFilter
+/// If no filter is set, all TaskTemplates will be returned
 ///
 
-message GetAllTaskTemplatesByWardRequest {
-  string ward_id = 1;
+message GetAllTaskTemplatesWithWardAndCreatorFilterRequest {
```

Comment:
not a scalable name, what if we need to filter by task count in the future? simply "GetTaskTemplates" is sufficient